SPEC.md

```diff
@@ -455,34 +670,72 @@ lines](https://www.unicode.org/versions/Unicode13.0.0/ch05.pdf):
 
 Note that for the purpose of new lines, CRLF is considered _a single newline_.
 
+### Disallowed Literal Code Points
+
+The following code points may not appear literally anywhere in the document.
+They may be represented in Strings (but not Raw Strings) using `\u{}`.
+
+* The codepoints `U+0000-0009`,
+    the codepoint `U+000B`,
+    or the codepoints `U+000E-001F`  (various control characters).
+* `U+007F` (the Delete control character).
+* Any codepoint that is not a [Unicode Scalar
+  Value](https://unicode.org/glossary/#unicode_scalar_value).
```

Comment:
This is U+D800 to U+DFFF (inclusive) then? Can we include that in the spec for convenience?